tests/test_270_005010X279A1.py

```diff
@@ -17,3 +17,94 @@ def test_270_dependent(x12_270_dependent_input, x12_270_dependent_transaction):
         assert model_result[0].x12() == x12_270_dependent_transaction
+
+
+def test_270_properties_subscriber_use_case(x12_270_subscriber_input):
+    """
+    Tests the Eligibility Inquiry property accessors with the subscriber use-case.
+    The subscriber use-case does not include a dependent record.
+    The input data fixture reflects conventional Eligibility usage where a transaction contains a single
+    information receiver, information source, subscriber, and optionally dependent.
+    """
+
+    with X12ModelReader(x12_270_subscriber_input) as r:
+        transaction_model = [m for m in r.models()][0]
+
+        information_source = transaction_model.information_source
+        assert isinstance(information_source, dict)
+        assert len(information_source) == 2
+        assert "hl_segment" in information_source
+        assert "loop_2100a" in information_source
+
+        information_receiver = transaction_model.information_receiver
+        assert isinstance(information_receiver, dict)
+        assert len(information_receiver) == 2
+        assert "hl_segment" in information_receiver
+        assert "loop_2100b" in information_receiver
+
+        subscriber = transaction_model.subscriber
+        assert isinstance(subscriber, dict)
+        assert len(subscriber) == 3
+        assert "hl_segment" in subscriber
+        assert "trn_segment" in subscriber
+        assert "loop_2100c" in subscriber
+
+        dependent = transaction_model.dependent
+        assert dependent is None
+
+
+def test_270_properties_dependent_use_case(x12_270_dependent_input):
+    """
+    Tests the Eligibility Inquiry property accessors with the dependent use-case.
+    The dependent use-case exercises all available properties.
+    The input data fixture reflects conventional Eligibility usage where a transaction contains a single
+    information receiver, information source, subscriber, and optionally dependent.
+    """
+
+    with X12ModelReader(x12_270_dependent_input) as r:
+        transaction_model = [m for m in r.models()][0]
+
+        information_source = transaction_model.information_source
+        assert len(information_source) == 2
+        assert "hl_segment" in information_source
+        assert "loop_2100a" in information_source
+
+        information_receiver = transaction_model.information_receiver
+        assert len(information_receiver) == 2
+        assert "hl_segment" in information_receiver
+        assert "loop_2100b" in information_receiver
+
+        subscriber = transaction_model.subscriber
+        assert len(subscriber) == 3
+        assert "hl_segment" in subscriber
+        assert "trn_segment" in subscriber
+        assert "loop_2100c" in subscriber
+
+        dependent = transaction_model.dependent
+        assert len(dependent) == 3
+        assert "hl_segment" in dependent
+        assert "trn_segment" in dependent
+        assert "loop_2100d" in dependent
+
+
+def test_property_usage(x12_270_subscriber_input):
```

Comment:
Hi @ccorley - I added a test case which uses the example we discussed.

Reply:
This is such a great example, I don't need to make the changes in my EDI fork that we discussed.  Nice!